appinventor/appengine/src/com/google/appinventor/client/editor/youngandroid/actions/ToggleConsoleAction.java

```diff
@@ -0,0 +1,12 @@
+package com.google.appinventor.client.editor.youngandroid.actions;
+
+import com.google.appinventor.client.Ode;
+import com.google.gwt.user.client.Command;
+
+public class ToggleConsoleAction implements Command {
+    @Override
+    public void execute() {
+        Ode ode = Ode.getInstance();
+        ode.setConsoleVisible(!ode.isConsoleVisible());
+    }
```

Comment:
We use the Google Java Style Guide, which has +2 indentation rather than +4. Reindent accordingly.